services/kaze/middlewares/falcon_guard.go

```diff
@@ -49,6 +52,10 @@ func FalconGuard() gin.HandlerFunc {
 				c.Abort()
 				return
 			}
+			var data map[string]interface{}
+			json.Unmarshal([]byte(user), &data)
```

Comment:
You should check for error handling here in the unmarshal part in the offchance falcon returns us an error instead of the required data